closure/goog/math/integer.js

```diff
@@ -566,6 +566,60 @@ goog.math.Integer.carry16_ = function(bits, index) {
   }
 };
 
+/**
+ * Returns this Integer divided by the given one.
+ * NOTE: Both this and the given Integer MUST be positive.
+ * @param {goog.math.Integer} other The Integer to divide this by.
```

Comment:
Yes, you're right, `divide` would crash and it's unfortunate it's set up that way :(. However, each method should be considered in isolation and abide by its annotated type contract. You never know how or where this method will be used in the future. 

The right thing to do here is to make both the `divide` and `slowDivide`'s `other` params non-nullable. Considering this would completely break people's JS if they passed null right now anyway, it should be a safe change. Can you do that? Thanks.